search-service/src/main/kotlin/com/egm/stellio/search/entity/service/EntityService.kt

```diff
@@ -74,57 +75,64 @@ class EntityService(
     suspend fun createEntity(
         ngsiLdEntity: NgsiLdEntity,
         expandedEntity: ExpandedEntity
-    ): Either<APIException, Unit> = either {
-        val sub = getSubFromSecurityContext()
+    ): Either<APIException, Unit> =
+        kotlin.runCatching {
```

Comment:
I find it easier to read if `kotlin.runCatching` is on the same line than the `Either<...>` (as you did for `deleteEntity`)